schema/rpc.fbs

```diff
@@ -223,24 +223,25 @@ enum SkeletonBone: uint8 {
     NONE = 0,
     HEAD = 1,
     NECK = 2,
-    CHEST = 3,
+    UPPER_CHEST = 3,
     CHEST_OFFSET = 4,
-    WAIST = 5,
-    HIP = 6,
-    HIP_OFFSET = 7,
-    HIPS_WIDTH = 8,
-    UPPER_LEG = 9,
-    LOWER_LEG = 10,
-    FOOT_LENGTH = 11,
-    FOOT_SHIFT = 12,
-    SKELETON_OFFSET = 13,
-    SHOULDERS_DISTANCE = 14,
-    SHOULDERS_WIDTH = 15,
-    UPPER_ARM = 16,
-    LOWER_ARM = 17,
-    HAND_Y = 18,
-    HAND_Z = 19,
-    ELBOW_OFFSET = 20,
+    CHEST = 5,
+    WAIST = 6,
+    HIP = 7,
+    HIP_OFFSET = 8,
+    HIPS_WIDTH = 9,
+    UPPER_LEG = 10,
+    LOWER_LEG = 11,
+    FOOT_LENGTH = 12,
+    FOOT_SHIFT = 13,
+    SKELETON_OFFSET = 14,
+    SHOULDERS_DISTANCE = 15,
+    SHOULDERS_WIDTH = 16,
+    UPPER_ARM = 17,
+    LOWER_ARM = 18,
+    HAND_Y = 19,
+    HAND_Z = 20,
+    ELBOW_OFFSET = 21,
```

Comment:
you're not supposed to change the IDs of the enum members.

Reply:
I would prefer to and it’s not 1.0

Reply:
No. Please don't change them.